contrib/python/pypodman/pypodman/lib/actions/commit_action.py

```diff
@@ -80,8 +82,16 @@ def commit(self):
                     flush=True)
                 return 1
             else:
-                ident = ctnr.commit(self.opts['image'][0], **self.opts)
-                print(ident)
+                ident = ctnr.commit(
+                    self.opts['image'][0],
+                    change=self.opts.get('change', None),
+                    message=self.opts.get('message', None),
+                    pause=self.opts['pause'],
+                    author=self.opts.get('author', None),
+                )
```

Comment:
Something special ``pause`` it doesn't get the "gentle" treatment too?

Reply:
pause is boolean type: always True or False in this case never None.